### ISSUE TITLE
[API endpoint] add a /v1/price

### ISSUE THREAD

**Issue by martindaniel4** [corradio/electricitymap#368]
This is following discussion with @corradio in https://github.com/corradio/electricitymap/issues/336

We will need an endpoint `/v1/price` that would require the parameters `countryCode` and `datetime` and return a JSON: 

See for reference `/v1/production`: 

- `http://electricitymap.org/v1/production?countryCode=FR&datetime=2017-02-01`
- Server - https://github.com/corradio/electricitymap/blob/master/web/server.js#L327

Here could be the response to:  http://electricitymap.org/v1/price?countryCode=FR&datetime=2017-02-01

```json
{
	"_id": "589124f993c96e62fce393c4",
	"countryCode": "FR",
	"datetime": "2017-01-31T23:45:00.000Z",
	"source": "rte-france.com",
	"price": 64.59,
	"createdAt": "2017-01-31T23:59:53.641Z"
}
```

**Comment by martindaniel4** [comment]
@corradio @nikkozzblu - can one of you take that one?

**Comment by martindaniel4** [comment]
Fixed in #369
